CHANGELOG.md

```diff
@@ -1,3 +1,32 @@
+## 2.1.0-beta.1
+ - feat: Deprecated endpoint methods are now being deprecated in generated client.
```

Comment:
Suggestion: Change to `Endpoint methods with Deprecated annotation are now also annotated in the client.`